### PR TITLE
remove void-live.rules if found

### PR DESCRIPTION
**Pull request by pfpulux**

### CODE REVIEW

**Comment by ahesford** [comment]
I am unfamiliar with the installer, but if this does what it seems, this is a much better approach to dropping the polkit rule than relying on INSTALL in base-files. That special-purpose cleanup ought to be dropped from base-files and live here where it is needed.

**Comment by q66** [comment]
the special purpose cleanup still needs to be there in order to fix people's installations made from older images

File: installer.sh.in [hunk 1]
```diff
@@ -1327,6 +1327,12 @@ ${BOLD}Do you want to continue?${RESET}" 20 80 || return
         unset USERLOGIN
     fi
 
+    POLKITRULE=$TARGETDIR/etc/polkit-1/rules.d/void-live.rules
+    if [ -f $POLKITRULE ]; then
+        rm -f $POLKITRULE
+    fi
```
Comment (ericonr) [inline installer.sh.in:1333]: since you're using `rm -f`, you can skip the conditional directly. (and then also skip using a variable, I guess)

**Comment by the-maldridge** [comment]
Per ericonr's suggestion please drop the conditional and use `rm -f` exclusively.

**Pull request closed without merging**